src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1123,11 +1123,12 @@ def get_az_rest_user_agent():
 
 def user_confirmation(message, yes=False):
     if yes:
-        return
+        return True
```

Comment:
I don't think you need to add a return value. As `False` will raise an exception, if code below where this function gets executed, the result for `user_confirmation()` must be `True`.

Reply:
Calling `core/util.py/user_confirmation()` does not use its return value. After adding the return value, it is exactly the same as the logic in `flexible_server_custom_common.py/user_confirmation()`, so they are merged